documents/functional_specifications/Functional Specification.md

```diff
@@ -71,35 +73,50 @@ Developed using Flutter 3, and commissioned by "We Are Evolution," this applicat
 - Allow both candidates and companies to create accounts.
 - Provide a like/dislike system for candidates to express interest in companies.
 - Include an administrative interface for companies to manage applications and communicate with candidates.
-- Jobseekers will be able to search for softskills through a search bar.
-And in case the softskills they want to add don't exist in the database, they can create it.
-- Allow companies to specify the soft skills they are looking for via checkboxes.
+- Jobseekers will be able to search for soft skills through a search bar.
+    - In case the soft skills they want to add don't exist in the database, they can create it.
+- Allow companies to specify the soft skills they are looking for via checkboxes or search bar.
 
 ### 2.3 User Roles
 
 **Intended users of the software:**
 
-- **Job Seekers:** Citizens seeking employment, whether skilled or low-skilled, by highlighting their soft skills rather than formal qualifications. The goal is to help job seekers find positions that match their soft skills.
+- **Job Seekers:** Citizens seeking employment, whether skilled or low-skilled, by highlighting their soft skills rather than formal qualifications. The goal is to help jobseekers find positions that match their soft skills.
 - **Companies:** Businesses of all sizes looking to attract candidates with the appropriate soft skills for available positions.
 
 ### 2.4 Use Cases for All Operations
 
 **Typical usage scenarios and tasks:**
 
-#### 2.4.1 Jobseekers Side
+### 2.4.1 Jobseekers Side
 
 Typical steps for jobseekers:
 
 1. Download the application.
 2. Select the preferred language for the application's text.
 3. Create an account by providing personal information.
-4. Select relevant soft skills via checkboxes.
+4. Select relevant soft skills via the search bar.
 5. Specify the geographical area for job searching.
 6. Access the homepage to view companies currently hiring for the selected soft skills.
-7. On the homepage the job seeker must like the company to initiate the match.
-8. If a company expresses interest in the jobseekers profile, the connection process begins.
+7. On the homepage the jobseeker must like the company to initiate the match.
+8. Then the jobseeker needs to wait for a company to start a conversation.
+
+**Example: Jobseeker's experience**
```

Comment:
This is the last comment I write about job-seekers, look throughout the file for this mistake.